manual/src/refman/expr.etex

```diff
@@ -195,6 +206,24 @@ is good style to use @'begin' \ldots 'end'@ inside control structures:
 \end{alltt}
 and @'(' \ldots ')'@ for the other grouping situations.
 
+\begin{caml_example}{toplevel}
+let x = 1 + 2 * 3
+let y = (1 + 2) * 3;;
+\end{caml_example}
+
+\begin{caml_example}{toplevel}
+let f a b =
+  if a = b then
+    print_string "Equal"
+  else
+    begin
+      print_string "Not Equal: ";
+      print_int a;
+      print_string " and ";
+      print_int b
```

Comment:
For a better display (and other print_* functions ...):
```
let f a b =
  if a = b then
    print_endline "Equal"
  else
    begin
      print_string "Not Equal: ";
      print_int a;
      print_string " and ";
      print_int b;
      print_newline ()
  end;;
```

Reply:
Both suggestions incorporated.